src/components/app/CompareHome.js

```diff
@@ -68,7 +74,9 @@ class CompareHomeImpl extends PureComponent<Props, State> {
             value={profile1}
           />
           <label className="compareHomeFormLabel" htmlFor="compareHomeProfile2">
-            Profile 2:
+            <Localized id="CompareHome--form-label-profile2">
+              Profile 2:
+            </Localized>
           </label>
           <input
             name="profile2"
```

Comment:
Hm, I wonder if we should localize the placeholder for these inputs. But I guess it's fine to leave them currently since they are only `http://` and that's universal for all locales.